### ISSUE TITLE
AWS Aurora Serverless via Data API

### ISSUE THREAD

**Issue by bkniffler** [hasura/graphql-engine#7303]
Hi,
Aurora Serverless support for Hasura Cloud or for Hasura on AppRunner via Data API would be really convenient. Would this be within Hasura scope?

**Comment by dnk8n** [comment]
I agree this could be very useful. Any way of abstracting the data API so that Hasura can sit on top of it just like conventional postgresql database infrastructure?
